pandas/core/groupby/groupby.py

```diff
@@ -172,7 +172,7 @@ class providing the base-class of operations.
     {examples}
     """)
 
-_pipe_template = """\
+_pipe_template = """
```

Comment:
Can you build the doc for pipe here to make sure this is OK? Just want to be sure that removing the continuation doesn't impact expected indentation during substitution

Reply:
This one seems OK.